UTM.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Please remove this file. It's not supposed to be in GIT because Xcode will auto-generate it for you.